app/policies/review_policy.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewPolicy
+  def initialize(book:, reviewer:)
+    @book, @reviewer = book, reviewer
+  end
+
+  def authorized_for_review?
+    if book_reviewers.include? reviewer
```

Comment:
Yes, I know what you mean. My current approach is that I will allow trivial one-liners in the AR model, but anything more complex gets extracted. Still means my models can get pretty long with lots of short methods though.

Anyway, your refactoring means this simpler now so I think it's fine to leave as is.